app/src/test/java/com/ikarabulut/GameSettingsTest.java

```diff
@@ -112,4 +119,30 @@ void setRuleSet_When2_Rules1Class() {
         MatcherAssert.assertThat(settings.getRules(), instanceOf(ComeAliveWithFour.class));
     }
 
+    @Test
+    @DisplayName("When prompted for rules and 2 is passed, then the rules carried out during the game will be the ComeAliveWithFour class")
+    void setRuleSet_When3_ComeAliveWithTwoClass() {
+        String input = "3";
```

Comment:
The test display name says "when 2 is passed" and the input here is "3".  Same for the test below.  I'm guessing some copy/pasta sauce. ;)